src/v2/Components/SavedSearchAlert/DownloadAppBanner.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import { Box, Flex, Text } from "@artsy/palette"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import {
+  Device,
+  DOWNLOAD_APP_URLS,
+  useDeviceDetection,
+} from "v2/Utils/Hooks/useDeviceDetection"
+import { useTracking } from "react-tracking"
+import {
+  ActionType,
+  ClickedAppDownload,
+  ContextModule,
+  OwnerType,
+} from "@artsy/cohesion"
+import { SavedSearchAttributes } from "../ArtworkFilter/SavedSearch/types"
+
+interface DownloadAppBannerProps {
+  savedSearchAttributes: SavedSearchAttributes
+}
+
+export const DownloadAppBanner: React.FC<DownloadAppBannerProps> = ({
+  savedSearchAttributes,
+}) => {
+  const tracking = useTracking()
+  const { device } = useDeviceDetection()
+
+  const handleClick = (platform: "iOS" | "Android") => {
+    let destinationPath
+    let subject
+
+    if (platform === "iOS") {
+      destinationPath = DOWNLOAD_APP_URLS[Device.iPhone]
+      subject = "Download on the App Store"
+    }
+
+    if (platform === "Android") {
+      destinationPath = DOWNLOAD_APP_URLS[Device.Android]
+      subject = "Download on the Google Play"
```

Comment:
#veryminor maybe this should be "Get it on Google Play" (to match the button text) or "Download on Google Play"? Not really sure it matters since this is just for analytics 🤷

Reply:
Done